### DISCUSSION TITLE
[SSG] rebuild the page when CMS content changes

### DISCUSSION THREAD

**Discussion by aamorozov** [vercel/next.js#14502] [Help]
What are the recommended strategies to do so when using headless cms(contentful, for example) to source the data from and use it for SSG? Triggering a rebuild of the project(`next build`) seems like the first option atm but this can be very time consuming

**Comment by balazsorban44** [comment]
You will like this 🙂https://github.com/vercel/next.js/discussions/11552

**Reply by aamorozov** [reply]
Awesome, exactly what i'm looking for! It seems like it's already supported by `unstable_revalidate` option in `getStaticProps` - is it assuming the usage of vercel or should be available everywhere?

**Reply by balazsorban44** [reply]
Using it on an app deployed to Azure, looks to be working fine. Even if it would have hiccups right now, it will definitely work with any platform when it gets stable, I'm sure. 🙂
